hooks/settings.py

```diff
@@ -28,10 +28,19 @@ def get_export_preset(self):
         
         :returns: Path on disk to Flame export preset 
         """
-        # base it on one of the default presets that ship with Flame.
-        return os.path.join(
-            self.parent.engine.export_presets_root,
-            "movie_file",
-            "QuickTime (H.264 720p 8Mbits).xml"
-        )
+        
+        if self.parent.engine.is_version_less_than("2018.1") :
```

Comment:
Lyne proposed to simply check if the new preset file exist instead of checking for flame version. This way, client could replace the export preset used for review in older flame version.